core/domain/recommendations_validators.py

```diff
@@ -87,15 +89,16 @@ def _validate_topic_similarities(cls, item):
                 to validate.
         """
 
-        all_topics = list(item.content.keys())
+        content = ast.literal_eval(item.content)
```

Comment:
Why we need to use `literal_eval` here?

Reply:
I have removed the literal_eval fix for now, once the reason for the error is confirmed we can fix the issue as you advised. Thanks.